<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SignalsSlotsApp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="appLabel">
        <property name="text">
         <string>Время</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="appBrowser"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pb_start">
        <property name="text">
         <string>Старт/Стоп</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_clear">
        <property name="text">
         <string>Очистить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_circle">
        <property name="text">
         <string>Круг</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>649</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_start</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_pb_start_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>-9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_clear</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_pb_clear_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>-9</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_pb_start_clicked()</slot>
  <slot>on_pb_clear_clicked()</slot>
  <slot>on_pb_circle_clicked()</slot>
 </slots>
</ui>
